### PR TITLE
chore(compute): add pagination samples

### PR DESCRIPTION
**Pull request by yuryu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/google-cloud-ruby/issues) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea.
- [ ] Follow the instructions in [CONTRIBUTING](CONTRIBUTING.md). Most importantly, ensure the tests and linter pass by running `bundle exec rake ci` in the gem subdirectory.
- [ ] Update code documentation if necessary.

Add pagination samples for the new compute api.

closes: #<issue_number_goes_here>

### CODE REVIEW

File: google-cloud-compute-v1/samples/pagination.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+#
+# @param [String] project project ID or project number of the Cloud project you want to list images from.
+def print_images_list project:
+  client = ::Google::Cloud::Compute::V1::Images::Rest::Client.new
+
+  # Make the request to list all non-deprecated images in a project.
+  request = {
+    project: project,
+    # max_results indicates the maximum number of items that will be returned per page.
+    max_results: 3,
```
Comment (rsamborski) [inline google-cloud-compute-v1/samples/pagination.rb:34]: Based on arguments provided by @amanda-tarafa in [this comment](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/1445/files#r687921807), please change to 100.

**Review by rsamborski — APPROVED** [review]
Small nit regarding number of MaxValues. LGTM besides that.

**Comment by yuryu** [comment]
Addressed @rsamborski's comment. Need review from @googleapis/yoshi-ruby before merging.

File: google-cloud-compute-v1/samples/acceptance/pagination_test.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "../pagination"
+require_relative "helper"
+
+
+class ComputeDefaultValuesTest < Minitest::Test
+  PROJECT = "windows-sql-cloud"
```
Comment (dazuma) [inline google-cloud-compute-v1/samples/acceptance/pagination_test.rb:20]: I assume successful test runs would require credentials for this project? Our CI is run against the project `helical-zone-771` (as in, the CI job has credentials for that project) so we probably need to use that project and set up test fixtures in that project instead. So the fact that CI appears to be running tests and succeeding, is puzzling to me. Can you explain what's going on?

Comment (rsamborski) [reply google-cloud-compute-v1/samples/acceptance/pagination_test.rb:20]: windows-sql-cloud is a special project which stores public SQL Server images. It should be available to any GCP project. PTAL here: https://cloud.google.com/compute/docs/images/os-details#sql_server

Comment (yuryu) [reply google-cloud-compute-v1/samples/acceptance/pagination_test.rb:20]: Yes, it should remain accessible and tests continue to pass. I used it following the python samples.

Comment (dazuma) [reply google-cloud-compute-v1/samples/acceptance/pagination_test.rb:20]: ok.

**Review by dazuma — APPROVED** [review]

**Comment by yuryu** [comment]
So again, I need someone to merge this 🙏

**Pull request merged**